midcli/gui/network/interface/create_update.py

```diff
@@ -20,15 +20,31 @@ class NetworkInterfaceCreate(Steps):
     method = StepsMethod.CREATE
 
     def step1(self, data):
-        return list(filter(None, [
-            Header("Interface Settings"),
-            Input("type") if self.method == StepsMethod.CREATE else None,
+        result = [Header("Interface Settings")]
+
+        if self.method == StepsMethod.CREATE:
+            result.append(Input("type"))
+
+        result.extend([
             Input("name"),
             Input("description"),
             Input("ipv4_dhcp"),
             Input("ipv6_auto"),
             Input("aliases", delegate=AliasesInputDelegate),
```

Comment:
This doesn't seem to enforce or handle the fact if a user provides an ip address without netmask information?